src/bin/cargo/commands/tree.rs

```diff
@@ -42,7 +42,14 @@ pub fn cli() -> Command {
             "SPEC",
             "Prune the given package from the display of the dependency tree",
         ))
-        .arg(opt("depth", "Maximum display depth of the dependency tree").value_name("DEPTH"))
+        .arg(
+            opt(
+                "depth",
+                "Maximum display depth of the dependency tree, \
+                or --depth=worktree to display dependencies from workspace members only",
```

Comment:
We should also update `cargo help tree`, see https://github.com/rust-lang/cargo/blob/master/src/doc/man/cargo-tree.md